Dockerfile

```diff
@@ -24,11 +20,10 @@ ENV LANGUAGE en_US:en
 ENV LC_ALL en_US.UTF-8
 
 # setup the environment
-WORKDIR /opt/superdesk/
+WORKDIR /opt/server/
 COPY ./docker/nginx.conf /etc/nginx/nginx.conf
 COPY ./docker/superdesk_vhost.conf /etc/nginx/sites-enabled/superdesk.conf
 COPY ./docker/start.sh /opt/superdesk/start.sh
```

Comment:
@dev-tsoni will this `/opt/superdesk/start.sh` after renaming the folder to `server/`?

Reply:
ah yes, will move the start script to server